pkg/handlers/generic/lifecycle/csi/nutanix-csi/handler.go

```
@@ -151,12 +151,17 @@ func (n *NutanixCSI) handleHelmAddonApply(
 		)
 	}
 
-	helmChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixStorageCSI)
+	storageChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixStorageCSI)
 	if err != nil {
-		return fmt.Errorf("failed to get values for nutanix-csi-config %w", err)
+		return fmt.Errorf("failed to get helm chart %q: %w", config.NutanixStorageCSI, err)
```

Great change to make it a little more "transactional" and fail fast